FixWhitespace.hs

```diff
@@ -150,16 +150,16 @@ main = do
 
       when verbose $ do
         putStrLn "Include whitelist:"
-        putStrLn (concatMap (++ "\n") incWhitelistPatterns)
+        putStrLn (unlines incWhitelistPatterns)
 
         putStrLn "Exclude blacklist:"
-        putStrLn (concatMap (++ "\n") excBlacklistPatterns)
+        putStrLn (unlines excBlacklistPatterns)
 
         putStrLn "Include:"
-        putStrLn (concatMap (++ "\n") incPatterns)
+        putStrLn (unlines incPatterns)
 
         putStrLn "Exclude:"
-        putStrLn (concatMap (++ "\n") excPatterns)
+        putStrLn (unlines excPatterns)
```

Comment:
This change is technically unrelated but pops up when running `hlint` from the Agda source root.